### ISSUE TITLE
APNS token type issue

### ISSUE THREAD

**Issue by Neustart** [arnesson/cordova-plugin-firebase#879]
File `AppDelegate+FirebasePlugin.m`.
In the method: `- (void)application:(UIApplication )application didRegisterForRemoteNotificationsWithDeviceToken:(NSData )deviceToken`
instead of
`[FIRMessaging messaging].APNSToken = deviceToken;`
I have to use
`[[FIRMessaging messaging] setAPNSToken:deviceToken type:FIRMessagingAPNSTokenTypeSandbox];`
every time when I making a build in debug mode (code signing iOS Developer).
Seems like issue appeared after update to the last Xcode (10.0).  It might be because APNS token type not properly defined because of that. Am I right ? Do you have any ideas ?
xCode: Version 10.0 (10A255)
cordova-plugin-firebase: version 2.0.1

**Comment by Hanzofm** [comment]
Same problem with this versions

**Comment by anaaChacon** [comment]
+1

**Comment by soumak77** [comment]
Check out this [cordova-ios issue](https://github.com/apache/cordova-ios/issues/407) to see if the workaround works to solve your issue

**Comment by christianguevara** [comment]
With that change, are you able to get notifications on iOS ?

**Comment by christianguevara** [comment]
Yes, already solved the problem updating to 2.0.4 and using legacy build system

**Comment by testanddeploy** [comment]
I just need to go --> File --> WorspaceSettings --> Build Settings(change here to "Legacy Build System"
